e2e/advanced/tekton_test.go

```diff
@@ -37,13 +41,50 @@ func TestTektonLikeBehavior(t *testing.T) {
 	t.Parallel()
 
 	WithNewTestNamespace(t, func(ctx context.Context, g *WithT, ns string) {
-		g.Expect(CreateOperatorServiceAccount(t, ctx, ns)).To(Succeed())
-		g.Expect(CreateOperatorRole(t, ctx, ns)).To(Succeed())
-		g.Expect(CreateOperatorRoleBinding(t, ctx, ns)).To(Succeed())
-
-		g.Eventually(OperatorPod(t, ctx, ns)).Should(BeNil())
-		g.Expect(CreateKamelPod(t, ctx, ns, "tekton-task", "install", "--skip-cluster-setup", "--olm=false", "--force")).To(Succeed())
+		operatorID := "camel-k-tekton-like"
 
+		g.Expect(CopyCamelCatalog(t, ctx, ns, operatorID)).To(Succeed())
```

Comment:
I think this is the case where we don't want to copy the catalog or the existing kits.